src/client/src/orangehrmPerformancePlugin/pages/ReviewList.vue

```diff
@@ -0,0 +1,313 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+<template>
+  <div class="orangehrm-background-container">
+    <oxd-table-filter :filter-title="$t('performance.review_list')">
+      <oxd-form @submitValid="filterItems" @reset="filterItems">
+        <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+          <oxd-grid-item>
+            <employee-autocomplete
+              v-model="filters.employee"
+              :params="{
+                includeEmployees: 'currentAndPast',
+              }"
+            />
+          </oxd-grid-item>
+          <oxd-grid-item>
+            <jobtitle-dropdown v-model="filters.jobTitle" />
+          </oxd-grid-item>
+          <oxd-grid-item>
+            <subunit-dropdown v-model="filters.subUnit" />
+          </oxd-grid-item>
+          <oxd-grid-item class="--offset-row-2">
+            <review-status-dropdown v-model="filters.status" />
+          </oxd-grid-item>
+          <oxd-grid-item class="--offset-row-2">
+            <date-input
+              v-model="filters.fromDate"
+              :label="$t('general.from_date')"
+            />
+          </oxd-grid-item>
+          <oxd-grid-item class="--offset-row-2">
+            <date-input
+              v-model="filters.toDate"
+              :label="$t('general.to_date')"
+            />
+          </oxd-grid-item>
+        </oxd-grid>
+
+        <oxd-divider />
+
+        <oxd-form-actions>
+          <oxd-button
+            display-type="ghost"
+            :label="$t('general.reset')"
+            type="reset"
+          />
+          <oxd-button
+            class="orangehrm-left-space"
+            display-type="secondary"
+            :label="$t('general.search')"
+            type="submit"
+          />
+        </oxd-form-actions>
+      </oxd-form>
+    </oxd-table-filter>
+    <br />
+    <div class="orangehrm-paper-container">
+      <table-header
+        :selected="0"
```

Comment:
selected hardcoded? :thinking: is it not selectable?

Reply:
yes not selectable in Review List page. only possible in Manage Review page

Reply:
Okay